<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Vijay" failures="1" tests="9" name="AdminModuleTestScripts.AdminModule" time="409.928" errors="0" timestamp="2023-08-10T18:19:59 IST" skipped="0">
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateUser" time="259.800"/>
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateTeamType" time="14.757"/>
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateTeamLevel" time="12.146">
    <failure message="expected [Level Number] but found [Don&#039;t Forget]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Level Number] but found [Don't Forget]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at AdminModuleTestScripts.AdminModule.CreateTeamLevel(AdminModule.java:81)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CreateTeamLevel -->
  <testcase classname="AdminModuleTestScripts.AdminModule" name="MapCountryLocation" time="18.803"/>
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateLocation" time="14.409"/>
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateDefaultRateCard" time="22.245"/>
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateServiceProvider" time="20.872"/>
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateNotification" time="20.856"/>
  <testcase classname="AdminModuleTestScripts.AdminModule" name="CreateActionDescription" time="26.040"/>
</testsuite> <!-- AdminModuleTestScripts.AdminModule -->
